ts/parts/Time.ts

```diff
@@ -744,7 +744,7 @@ Highcharts.Time.prototype = {
         this: Highcharts.Time,
         f: (string|Array<T>|Highcharts.Dictionary<T>)
     ): Highcharts.Dictionary<T> {
-        if (!isObject(f, true)) { // check for string or array
+        if (!isObject(f)) { // check for string or array
```

Comment:
This now changed functionality from checking that f is not an object, to checking that f is neither an object nor an array.

Reply:
Otherwise the purpose of the function to output only dictionaries would not work.